firmware/robot2015/src-fpga/robocup2015.ucf

```diff
@@ -0,0 +1,27 @@
+# This is the "User Constraints File" for configuring the fpga
```

Comment:
robocup2015.ucf is the one Doho used for testing. robocup.ucf is the one from 2011. Then again, I made the new ucf file for the 2015 stuff today, so they'll both be replaced by that one pretty soon. I left the original one there so it would still generate a bitfile correctly during the build.